src/components/widgets/filesystem/FileEditorTextOnly.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <textarea
+    class="editor v-input v-textarea theme--dark"
```

Comment:
looks like i forgot to add this comment: i think line starts are kinda hard to read on mobile, so a padding would be good here (just like monaco has)
```suggestion
    class="editor v-input v-textarea theme--dark px-2"
```

other than that it lgtm :)